src/main/kotlin/inc/evil/d17map/dtos/ReservationRequest.kt

```diff
@@ -3,33 +3,53 @@ package inc.evil.d17map.dtos
 import com.fasterxml.jackson.annotation.JsonFormat
 import inc.evil.d17map.enums.ReservationType
 import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.*
 import java.time.LocalDate
 import java.time.LocalTime
 import java.util.*
 
 data class ReservationRequest(
+    @field:NotBlank(message = "Title must not be blank.")
     val title: String,
+
+    @field:NotBlank(message = "Description must not be blank.")
     val description: String,
 
     @Schema(type = "string", format = "date")
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    @field:NotNull(message = "Date must not be null.")
+    @field:FutureOrPresent(message = "Date must be today or in the future.")
     val date: LocalDate,
 
     @Schema(type = "string", format = "time", example = "11:30")
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm")
+    @field:NotNull(message = "Start time must not be null.")
+    @field:FutureOrPresent(message = "Start time must be now or in the future.")
```

Comment:
Tutaj jest ryzyko, że jak chcemy zarezerwować na przyszły dzień ale godzinę wcześniejszą niż obecna, to nam wyrzuci błąd (chyba). Czy ta adnotacja działa poprawnie z typem `LocalTime`¿

Reply:
Racja - przy LocalTime to nie zadziała, musiałoby być LocalDateTime